cmd/podman/play/kube.go

```diff
@@ -84,7 +84,18 @@ func kube(cmd *cobra.Command, args []string) error {
 		fmt.Fprintf(os.Stderr, l)
 	}
 
-	fmt.Printf("Pod:\n%s\n", report.Pod)
+	switch len(report.Pods) {
+	case 0:
+		return nil
```

Comment:
Hm. Are there cases where containers could be made, but not pods? We might want to make 0 print nothing but not return yet

Reply:
I actually disagree, there really shouldn't be a case containers are created and pods are not. I think return early here if no printing is needed